src/containers/DataPackageView.js

```diff
@@ -18,8 +19,12 @@ export class DataPackageViewContainer extends React.Component {
   }
 
   componentDidMount(){
-    let url = "https://raw.githubusercontent.com/anuveyatsu/test_data/master/datapackage.json";
+    let baseUrl = "https://bits.staging.datapackaged.com/metadata/";
+    let url = baseUrl + this.props.publisherName + "/" + this.props.packageName + "/_v/latest/datapackage.json";
+    // let url = "https://raw.githubusercontent.com/anuveyatsu/test_data/master/datapackage.json";
     this.props.dataPackageActions.getDataPackage(url);
+    this.props.dataPackageActions
+      .getDataPackageMetaData(this.props.publisherName, this.props.packageName);
```

Comment:
Why do you need to fetch metadata via our api agian? It just happened in the line above, by calling `this.props.dataPackageActions.getDataPackage(url);`